src/startupscriptgenerator/src/node/scriptgenerator.go

```diff
@@ -52,15 +53,12 @@ func (gen *NodeStartupScriptGenerator) GenerateEntrypointScript() string {
 	scriptBuilder.WriteString("#!/bin/sh\n")
 	scriptBuilder.WriteString("\n# Enter the source directory to make sure the script runs where the user expects\n")
 	scriptBuilder.WriteString("cd " + gen.SourcePath + "\n\n")
-	scriptBuilder.WriteString("if [ -f ./" + consts.BuildManifestFileName + " ]; then\n")
-	scriptBuilder.WriteString("    echo \"Found '" + consts.BuildManifestFileName + "'\"\n")
-	scriptBuilder.WriteString("    . ./" + consts.BuildManifestFileName + "\n")
-	scriptBuilder.WriteString("fi\n\n")
 
 	// Expose the port so that a custom command can use it if needed.
 	common.SetEnvironmentVariableInScript(&scriptBuilder, "PORT", gen.BindPort, DefaultBindPort)
 
 	if !gen.SkipNodeModulesExtraction {
+		scriptBuilder.WriteString("compressedNodeModulesFile=\"" + gen.Manifest.CompressedNodeModulesFile + "\"\n")
```

Comment:
Filed an issue now to fix this as we can minimize the amount of script that we generate by doing majority of logic in go